apps/explorer/src/hooks/useErc20.ts

```diff
@@ -79,9 +71,38 @@ export function useMultipleErc20(
     [addresses, tokenListTokens]
   )
 
+  // If native token is in the list of tokens to be fetched, memoize it here
+  const native = useMemo(
+    () =>
+      addresses.reduce<Record<string, TokenErc20> | undefined>((native, address) => {
+        if (native) return native
```

Comment:
There is a const with name `native` and the function parameter with the same name, will be easier to read the code with different names